tests/MenderAPI/authentication.py

```diff
@@ -108,12 +107,6 @@ def _create_tenant(self, username):
                                                                                              username)
         return docker_compose_cmd(cmd)
 
-    def _create_tenant_user(self, email, tenant_id):
```

Comment:
Is this called in one place only?

Reply:
to my knowledge, yes, grepped through everything